docs/graph.md

```diff
@@ -67,6 +67,30 @@ Edges can be added between the nodes as follows:
 </edge>
 ```
 
+## Creating graphs
```

Comment:
Should we add these new commands to warcli.md as well? Or are we getting in to maintenence hell where every new rpc requires multiple doc updates

Reply:
Let's not define define commands in two places :)

Can markdown import other markdown? If so we could document each subcommand in one file and import it in two places, but otherwise my preference would be not to...

Reply:
seems like Click should be able to auto-document. thatd be nice. Oterwise i dunno the best way to organize. maybe warcli.md has most commands but groups like `scenarios` and `graph` we can link to the individual doc pages instead ?